src/usdb_syncer/song_loader.py

```diff
@@ -82,8 +82,10 @@ class Context:
     @classmethod
     def new(
         cls, details: SongDetails, options: Options, info: DownloadInfo, logger: Log
-    ) -> Context:
-        txt_str = usdb_scraper.get_notes(details.song_id, logger)
+    ) -> Union[Context, None]:
```

Comment:
In Python 3.10 you generally don't need `Union`. You can write `Context | None` instead.

Reply:
Thanks for the suggestion, I didn’t know that.